src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -81,6 +81,61 @@ public void canPutAndGet() {
     assertThat(getRulesText(), is(rule));
   }
 
+  @Test
+  public void canUpdateCirculationRuleInDifferentPolicyOrder() {
+
+    UUID lp1 = UUID.randomUUID();
+    UUID rp1 = UUID.randomUUID();
+    UUID np1 = UUID.randomUUID();
+    UUID op1 = UUID.randomUUID();
+    UUID ip1 = UUID.randomUUID();
```

Comment:
Can these IDs have more meaningful names? And indexing them with `1` does not make much sense since we only have one of each.